openapi/src/main/scala/sttp/livestub/openapi/OpenapiModels.scala

```diff
@@ -126,10 +133,11 @@ object OpenapiModels {
       schema <- c.as[Map[String, (String, List[OpenapiResponseContent])]]
     } yield {
       schema.collect {
-        case (code, (desc, content)) if code != "default" =>
-          OpenapiResponse(ResponseStatusCode.Fixed(StatusCode.unsafeApply(code.toInt)), desc, content)
-        case (code, (desc, content)) if code != "default" =>
+        case (code, (desc, content)) if code == "default" =>
           OpenapiResponse(ResponseStatusCode.Default, desc, content)
+        case (IsStatusCode(code), (desc, content)) =>
```

Comment:
Made this the second case for perf reasons since the case of it being "default" will cause a `throw` in the `Try` block which we'd like to avoid since `default` will be pretty common.